lib/meta_inspector/parsers/images.rb

```diff
@@ -3,13 +3,22 @@
 module MetaInspector
   module Parsers
     class ImagesParser < Base
+      DEFAULT_IMG_BLACKLIST =  ['.woff', '.font', '.tiff', '.tif', 'data:'].freeze
+
       delegate [:parsed, :meta, :base_url]         => :@main_parser
       delegate [:each, :length, :size, :[], :last] => :images_collection
 
       include Enumerable
 
       def initialize(main_parser, options = {})
         @download_images = options[:download_images]
+
+        download_image_options = options[:download_image_options] || { fetch_all_image_meta: false, image_blacklist_words: DEFAULT_IMG_BLACKLIST }
+        @fetch_all_image_meta = download_image_options[:fetch_all_image_meta]
+        @image_blacklist_words =  download_image_options[:image_blacklist_words]
+        @max_image_downloads =  download_image_options[:max_image_downloads] # Default's to nil and if nil will not remove any photos
+        @reject_invalid_images = download_image_options[:reject_invalid_images]
```

Comment:
We may be approaching too many options such that the gem behaves in too many different ways ... so if anyone has input on how to handle this that'd be great.  I'm trying to not change any existing behavior to avoid regressions for people, but perhaps we need to make some of these things the new behavior instead of options you can specify.